### PR TITLE
Initial Maestro Q/A tests & common utility flows

### PR DESCRIPTION
**Pull request by itayplav**
The following Maestro Q/A automation tests are added:
| Name 	| Expected Result 	|
|---	|---	|
| C310-launch-no-crash.yaml 	| User is able to open Edge without crashing 	|
| C311-relaunch-no-crash.yaml 	| Edge should not crash when relaunching the App 	|
| ~C314-login-no-2fa.yaml~ 	| ~User is able to login with a valid Username and Password without 2FA or IP2FA blocked.~ 	|
| ~C431-create-account.yaml~ 	| ~Tapping "Create" takes user to new Wallet List Scene with: BTC, ETH, LTC, BCH, DASH.~ 	|
| ~C753-login-2fa.yaml~ 	| ~User is able to login with a valid Username + Password + Backup Code~ 	|

EDIT: The final three tests in the above table have been removed from this PR as they require refactoring due to UI changes. 

Additionally this PR sets up the:
- `./maestro` subfolder
- `./maestro/env.js` account credential store (added to `.gitignore`)
- primary `entrypoint.yaml` with subfolders used to group tests, each having their own `entrypoint.yaml` (currently only `./maestro/02-launch-login/`)

<video src="https://user-images.githubusercontent.com/4023066/223303035-05f0ecd9-14f6-4aa6-9cc0-07f719ec3c1f.mp4" width="500"></video>

### Requirements

- [ ] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [x] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203878358693337

### CODE REVIEW

File: maestro/02-launch-login/C311-relaunch-no-crash.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+---
+- runFlow:
+    file: ../common/launch.yaml
+
+- pressKey: home
+
+- runFlow:
+    file: ../common/launch.yaml
+
+
```
Comment (swansontec) [inline maestro/02-launch-login/C311-relaunch-no-crash.yaml:19]: Does prettier or some other tool support YAML? Can we get an auto-formatter in here to ensure things like this don't happen?

Comment (swansontec) [reply maestro/02-launch-login/C311-relaunch-no-crash.yaml:19]: Yep! https://prettier.io/blog/2018/07/29/1.14.0.html

This needs to be tested, but I think it's a good starting point for our integration:

```json
{
  "scripts": {
    "fix": "eslint . --fix && yarn-deduplicate && prettier -w maestro/**/*.yaml"
    "lint": "eslint . && prettier maestro/**/*.yaml"
  },
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": "eslint",
    "*.yaml": "prettier -w"
  }
}
```

File: maestro/env-sample.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Define test specific accounts here.
+// eslint-disable-next-line no-undef
+output.logins = {
+  C314: { USERNAME: '', PASSWORD: '', PIN: '', BACKUP_CODE: '' },
+  C753: { USERNAME: '', PASSWORD: '', PIN: '', BACKUP_CODE: '' }
```
Comment (swansontec) [inline maestro/env-sample.js:5]: I don't know how your sidebar with RJ went. These names could be hard to work with. I could see using names like "noPinUser", "no2faUser", "eosUser", "unactivatedEosUser", etc. being easier long-term, but like I said, this is really up to RJ about how he wants the workflow to go.

**Review by swansontec — CHANGES_REQUESTED** [review]
This looks really good!

File: maestro/env-sample.js (hunk 2 above)
Comment (itayplav) [reply maestro/env-sample.js]: As discussed, the test file name containing the test description (eg `C311-relaunch-no-crash.yaml`) means we do not need to add further verbiage to the property names of the `output.logins.<name>...` for credentials to maintain readability. In either case, the scope of this PR has been reduced to just the first two tests (`C310-launch-no-crash.yaml` & `C311-relaunch-no-crash.yaml`).

File: .gitignore [hunk 3]
```diff
@@ -110,3 +110,6 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+
+# Maestro
+maestro/env.js
```
Comment (swansontec) [inline .gitignore:115]: Put a leading slash here. `/maestro/env.js`. That makes it relative to the repository root.

File: maestro/02-launch-login/C311-relaunch-no-crash.yaml (hunk 1 above)
Comment (swansontec) [inline maestro/02-launch-login/C311-relaunch-no-crash.yaml:19]: Extra blank line.

**Review by swansontec — APPROVED** [review]

**Review by swansontec — APPROVED** [review]

**Pull request merged**